apps/document-check-function/check-new-document/clam-av-client.js

```diff
@@ -0,0 +1,19 @@
+import NodeClam from 'clamscan';
+
+/**
+ * @param {import('node:stream').Readable} blobStream
+ * @returns {Promise<boolean>}
+ */
+export const scanStream = async (blobStream) => {
+	const clamScan = await new NodeClam().init({
+		debugMode: true,
+		clamdscan: {
+			host: '0.0.0.0',
```

Comment:
I guess this is supposed to come from some config?

Reply:
Nice catch yes